File: Java/Missing_Number.java
void main(){
    int[] nums1 = {9,6,4,2,3,5,7,0,1};
    Arrays.sort(nums1);
    int[] nums = java.util.Arrays.copyOf(nums1, nums1.length + 1);
    nums[nums.length - 1] = 0;
    IO.println(Arrays.toString(nums));

    for (int i = 0; i < nums.length; i++) {
        if (i!=nums[i]) {
            IO.println(i);
            break;
        }
    }

}

File: Python/Missing_Number.py
def missing_number():
    nums1 = [9, 6, 4, 2, 3, 5, 7, 0, 1]
    nums1.sort()
    
    nums = nums1 + [0]
    print(nums)
    
    for i in range(len(nums)):
        if i != nums[i]:
            print(i)
            break

if __name__ == "__main__":
    missing_number()
